package.json

```diff
@@ -42,16 +42,19 @@
 		"heroku-postbuild": "yarn build"
 	},
 	"dependencies": {
```

Comment:
Note for this, if a child package sets a peer dependency, yarn expects the root package.json to include that dependency.